terraform/modules/aws-brig-prekey-lock-event-queue/README.md

```diff
@@ -0,0 +1,28 @@
+Terraform module: Brig pre-key locking & event queue 
+====================================================
+
+State: __experimental__
+
+This module allows Brig to leverage AWS resources (A) to lock pre-keys during insertion and
+retrieval, and (B) to establish a message queue for internal events.
```

Comment:
```suggestion
This module allows wire-server's brig service to leverage AWS resources (A) to acquire a lock using dynamoDB (used during insertion and retrieval of prekeys in cassandra to avoid race conditions), and (B) to establish a message queue for internal events (used e.g. during user deletions).
```